docs/guidelines/language/operational-emails/reports-and-data-provisioning-emails.md

```
@@ -0,0 +1,121 @@
+---
+sidebar_position: 5
+sidebar_label: Reports and data provisioning emails
+title: Reports and data provisioning emails
+doc-type: 'banner'
+description: 'Well-written notifications about the completion and availability of generated reports or imported data ensure stakeholders can act quickly and confidently.'
+---
+
+#
+
+Note: Use these rules alongside the [General rules](#) and we have used [App name] as the standard placeholder for you to adapt individually to your product. All placeholders are shown with square brackets [placeholder].
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)